package-scripts.yml

```diff
@@ -0,0 +1,48 @@
+scripts:
```

Comment:
Hm. So we'd just have `package-scripts.js`, but it could either have YAML or JS inside, and the script should figure out what's what?